azure/postgres_firewall.sh

```diff
@@ -0,0 +1,72 @@
+# Postgres Servers firewall related functions
+
+# azure_postgres_firewall_rule_create creates a new rule
+# for the firewall of a server of "managed Postgres".
+# `name` is the name of the rule.
+# `group` is the name of resource group.
+# `servername` is the name of the Postgres server.
+# `startip` is the start IP address of the firewall rule. (IPV4 format)
+# `endip` is the end IP address of the firewall rule. (IPV4 format)
```

Comment:
The docs do not say what it returns when succeed and I cannot figure this out by looking at the code either.